### PR TITLE
libwpe/cog: version bump to latest stable releases

### PR DESCRIPTION
**Pull request by Bastian-Krause**
- libwpe: version bump 1.8.0 -> 1.10.0
- cog: version bump 0.8.0 -> 0.8.1

Merged patches were dropped, SRCREV_class-devupstream was set to the latest commitishes on the respective master branches.

### CODE REVIEW

File: recipes-browser/cog/cog_0.8.0.bb [hunk 1]
```diff
@@ -1,14 +0,0 @@
-require cog.inc
-require conf/include/devupstream.inc
-
-SRC_URI = "https://github.com/Igalia/cog/releases/download/v${PV}/cog-${PV}.tar.xz \
-           file://0001-platform-add-a-common-EGL-proc-address-loader-with-d.patch \
-           file://0002-egl-proc-address.h-add-a-license-header.patch \
```
Comment (philn) [inline recipes-browser/cog/cog_0.8.0.bb:6]: Can the files be removed as well?

**Review by philn — CHANGES_REQUESTED** [review]

File: recipes-browser/cog/cog_0.8.0.bb (hunk 1 above)
Comment (Bastian-Krause) [reply recipes-browser/cog/cog_0.8.0.bb:6]: Oh of course, sorry about that.

**Comment by Bastian-Krause** [comment]
Force-pushed:
- also dropped those cog patch files

**Review by philn — APPROVED** [review]

**Comment by clopez** [comment]
Thanks!

**Pull request merged**